lms/views/dashboard.py

```diff
@@ -79,3 +68,28 @@ def api_assignment_stats(self):
             }
             for s in stats
         ]
+
+    def get_request_assignment(self):
+        assignment = self.assignment_service.get_by_id(self.request.matchdict["id_"])
+        if not assignment:
+            raise HTTPNotFound()
+
+        if not self.assignment_service.is_member(assignment, self.request.user):
+            raise HTTPUnauthorized()
+
+        return assignment
+
+    def _set_lti_user_cookie(self, response):
+        auth_token = (
+            BearerTokenSchema(self.request).authorization_param(self.request.lti_user)
+            # White space is not allowed as a cookie character, remove the leading part
+            .replace("Bearer ", "")
+        )
+        response.set_cookie(
+            "authorization",
+            value=auth_token,
+            secure=not self.request.registry.settings["dev"],
+            httponly=True,
+            max_age=60 * 60 * 24,  # 24 hours, matches the lifetime of the auth_token
```

Comment:
I reckon it's fine to match these duration, the LTI user was just created in the line above.